docs/advanced/math-utils.mdx

```diff
@@ -1,5 +1,6 @@
 ---
 title: Math Utils
+description: The `@tscircuit/math-utils` package provides a set of utilities that are
```

Comment:
I dont think descriptions should have incomplete descriptions, even if it goes over the limit